### DISCUSSION TITLE
Mixing packaged extensions and non-packaged ones

### DISCUSSION THREAD

**Discussion by j3k0** [airsdk/apm#12] [Ideas]
So excited about this tool. Can't wait to see it get production ready!

I tried to figure out from the documentation: is it still possible to use ANEs that have not been packaged with `apm`. That would be nice, to smooth the transition from our current system to using `apm`.

Along the same line, is it possible for a project to define additional manifest / info additions (that `apm` will merge with its own in the generated app-descriptor)?

Maybe a solution for that is to create a pseudo package with the other configurations needed by the app? Can it be used without being published?

**Comment by marchbold** [comment]
Yes definitely possible to use existing extensions.

The output from the tool places all extensions in a directory for you to add to your ide or build scripts and you can add other extensions there or leave them wherever you currently have them

With manifest additions and info additions etc there are files that you can provide the client that will get merged with the packages. These are designed for specific configuration additions or accounting for extensions that aren't in a package 

https://github.com/airsdk/apm/wiki/Usage-Generate#advanced-configuration

**Reply by j3k0** [reply]
I missed that in the doc, that's perfect!

**Reply by marchbold** [reply]
It's just my first take on it, so if you have any suggestions please let me know
